Algorithmia/client.py

```diff
@@ -32,20 +35,23 @@ def __init__(self, apiKey = None, apiAddress = None, caCert = None):
             caCert = os.environ.get('REQUESTS_CA_BUNDLE')
             self.catCerts(caCert)
             self.requestSession.verify = self.ca_cert
+            atexit.register(self.exit_handler)
```

Comment:
Could move the atexit handler into catCerts? Then wouldn't need to have it in all three branches.